api/v1alpha1/groupversion_info.go

```diff
@@ -15,8 +15,8 @@ limitations under the License.
 */
 
 // Package v1alpha1 contains API Schema definitions for the misf v1alpha1 API group
-//+kubebuilder:object:generate=true
-//+groupName=misf.ibm.com
+// +kubebuilder:object:generate=true
+// +groupName=misf.ibm.com
```

Comment:
Why do we need this change?

Reply:
wasn't this decided to not keep any whitespace between the comment slashes?

Reply:
Why in this PR?
I checked other kubebuilder markers and we don't have a space in between.

Reply:
Sure, I will remove that specific commit

Reply:
Removed the lint commit